### DISCUSSION TITLE
Flowbite modal is not working in React

### DISCUSSION THREAD

**Discussion by MarioStipetic** [themesberg/flowbite#318] [General]
Hi, I want to use the regular Flowbite CSS code  inside my React app.
https://flowbite.com/docs/components/modal/
For example:
1. I added the following code inside my react file:
https://flowbite.com/docs/components/modal/#default-modal

3. I added the CSS in the header:
` <link rel="stylesheet" href="https://unpkg.com/flowbite@1.5.3/dist/flowbite.min.css" />`

4. And include the following JavaScript file before the end of the body element:
`<script src="https://unpkg.com/flowbite@1.5.3/dist/flowbite.js"></script>`

When I include the mentioned code from 1., the modal just do not work.
How come this happens, can I fix it somehow?

Flowbite-react library is too complex for me, does not have any documentation, and also do not have a lot of templates available as the the regular Flowbite (not flowbite -react).

**Comment by MarioStipetic** [comment]
So, when I add the modal inside index.html that is located in the react root, the modal works.
Then I add the https://flowbite.com/docs/components/modal/#default-modal mode code   inside some react function, the modal does not work.

**Comment by cupid20103** [comment]
https://www.flowbite-react.com/docs/components/modal

Please check this and follow it 💯
